Neos.ContentRepository.Core/Classes/EventStore/EventsToPublishFailed.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neos\ContentRepository\Core\EventStore;
+
+use Neos\EventStore\Exception\ConcurrencyException;
+use Neos\EventStore\Model\EventStream\ExpectedVersion;
+
+/**
+ * @internal
+ */
+final readonly class EventsToPublishFailed
```

Comment:
That is a weird name, how can the "events to publish" fail?
Why do we need a custom exception at all? can't we just forward the `ConcurrencyException`?

Reply:
I think so too...

Reply:
I have locally added the versions directly on the exception, that _would_ be neat I think but needs a chnage in the eventstore package, so a bit convoluted to get it all together...

Reply:
we now rethrow with ->throw and removed this class